src/views/Login.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+    <div class="grid grid-cols-2 login">
+        <div class="left_content">
+            <div class="logo-area">
+                <img class="h-14 mt-16 ml-14" src="../assets/views/securious-logo.svg" alt="Securious"/>
+            </div>
+            <div class="login-form mt-24 max-w-sm flex flex-col ml-40">
+                <div class="form-title">Einloggen</div>
+                <Form class="flex flex-col space-y-3 mt-4" @submit="login">
+                    <Field class="inp rounded-none p-2 text-xs" name="email" type="email" placeholder="Ihre@email.com" required />
```

Comment:
Looks like Vee-validate validation is not being used. Either fix validation or simply add manual validation.